components/MoneyInput/MoneyInput.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from '@testing-library/react'
+import { MoneyInput } from '.'
+
+describe('Modal component', () => {
+    it('renders correctly', () => {
+        render(
+            <MoneyInput name='input' label='label'/>
```

Comment:
You could also test if the inputted value is changed when the user types anything